python/pyspark/ml/regression.py

```diff
@@ -1740,7 +1742,7 @@ class AFTSurvivalRegression(_JavaRegressor, _AFTSurvivalRegressionParams,
     @keyword_only
     def __init__(self, *, featuresCol="features", labelCol="label", predictionCol="prediction",
                  fitIntercept=True, maxIter=100, tol=1E-6, censorCol="censor",
-                 quantileProbabilities=list([0.01, 0.05, 0.1, 0.25, 0.5, 0.75, 0.9, 0.95, 0.99]),
+                 quantileProbabilities=DEFAULT_QUANTILE_PROBABILITIES,
```

Comment:
This is still a mutable default, isn't it? Shouldn't `DEFAULT_QUANTILE_PROBABILITIES` by a `tuple`, if we want to be consistent?

Reply:
Bugbear doesn't complain, but you're right:
```
MacBook-Pro-van-Fokko:spark fokkodriesprong$ python3
Python 3.6.6 (v3.6.6:4cf1f54eb7, Jun 26 2018, 19:50:54) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from typing import List
>>> 
>>> DEFAULT_QUANTILE_PROBABILITIES = [0.01, 0.1, 0.99]
>>> 
>>> def mutate_arguments(mutate: List = DEFAULT_QUANTILE_PROBABILITIES):
...   mutate.append(22)
...   print(mutate)
... 
>>> mutate_arguments()
[0.01, 0.1, 0.99, 22]
>>> mutate_arguments()
[0.01, 0.1, 0.99, 22, 22]
>>> mutate_arguments()
[0.01, 0.1, 0.99, 22, 22, 22]
>>> mutate_arguments()
[0.01, 0.1, 0.99, 22, 22, 22, 22]
```